libs/cache/file_cache_env_test.go

```diff
@@ -0,0 +1,191 @@
+package cache
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"sync/atomic"
+	"testing"
+
+	"github.com/databricks/cli/libs/env"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCacheEnabledEnvVar(t *testing.T) {
+	ctx := context.Background()
+	tempDir := t.TempDir()
+
+	tests := []struct {
+		name         string
+		envValue     string
+		expectCached bool
+	}{
+		{
+			name:         "cache enabled with 'true'",
+			envValue:     "true",
+			expectCached: true,
+		},
+		{
+			name:         "cache disabled with 'false'",
+			envValue:     "false",
+			expectCached: false,
+		},
+		{
+			name:         "cache disabled when empty",
+			envValue:     "",
+			expectCached: false,
+		},
+		{
+			name:         "cache disabled with invalid value",
+			envValue:     "yes",
+			expectCached: false,
+		},
+		{
+			name:         "cache disabled with '1'",
+			envValue:     "1",
+			expectCached: false,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// Set up environment
+			if tt.envValue != "" {
+				t.Setenv("DATABRICKS_CACHE_ENABLED", tt.envValue)
+			} else {
+				os.Unsetenv("DATABRICKS_CACHE_ENABLED")
```

Comment:
If you need this to always be cleared before the test, you can run this outside the for loop.